aten/src/ATen/native/native_functions.yaml

```diff
@@ -10960,20 +10954,26 @@
 - func: linalg_matrix_norm.str_ord_out(Tensor self, str ord='fro', int[] dim=[-2,-1], bool keepdim=False, *, ScalarType? dtype=None, Tensor(a!) out) -> Tensor(a!)
   python_module: linalg
 
-- func: linalg_svd.U(Tensor self, bool full_matrices=True, *, Tensor(a!) U, Tensor(b!) S, Tensor(c!) Vh) -> (Tensor(a!) U, Tensor(b!) S, Tensor(c!) Vh)
+- func: linalg_svd(Tensor A, bool full_matrices=True) -> (Tensor U, Tensor S, Tensor Vh)
   python_module: linalg
+  variants: function
+  structured_delegate: linalg_svd.out
 
-- func: linalg_svd(Tensor self, bool full_matrices=True) -> (Tensor U, Tensor S, Tensor Vh)
+- func: linalg_svd.out(Tensor A, bool full_matrices=True, *, Tensor(a!) U, Tensor(b!) S, Tensor(c!) Vh) -> (Tensor(a!) U, Tensor(b!) S, Tensor(c!) Vh)
```

Comment:
This is most likely a breaking change. Please keep it as `linalg_svd.U`.
See the section about overloads
https://github.com/pytorch/pytorch/blob/60632a00fe6212159164d92ad69766925ab15c97/aten/src/ATen/native/README.md?plain=1#L170-L171
Using `.U` is recommended overload name and similar naming is used for other out variants.
> you can use the name or type of the first differing argument as an overload name.